src/pages/SetPasswordPage.js

```diff
@@ -1,135 +0,0 @@
-import React, {Component} from 'react';
-import {
-    SafeAreaView,
-    Text,
-    TextInput,
-    View,
-} from 'react-native';
-import PropTypes from 'prop-types';
-import {withOnyx} from 'react-native-onyx';
-import _ from 'underscore';
-import lodashGet from 'lodash.get';
-import styles from '../styles/styles';
-import ExpensifyCashLogo from '../../assets/images/expensify-cash.svg';
-import {setPassword} from '../libs/actions/Session';
-import ONYXKEYS from '../ONYXKEYS';
-import variables from '../styles/variables';
-import ButtonWithLoader from '../components/ButtonWithLoader';
-
-const propTypes = {
-    /* Onyx Props */
-
-    // The details about the account that the user is signing in with
-    account: PropTypes.shape({
-        // An error message to display to the user
-        error: PropTypes.string,
-
-        // Whether or not a sign on form is loading (being submitted)
-        loading: PropTypes.bool,
-    }),
-
-    // The credentials of the logged in person
-    credentials: PropTypes.shape({
-        // The email the user logged in with
-        login: PropTypes.string,
-
-        // The password used to log in the user
-        password: PropTypes.string,
-    }),
-
-    route: PropTypes.shape({
-        params: PropTypes.shape({
-            validateCode: PropTypes.string,
-        }),
-    }),
-};
-
-const defaultProps = {
-    account: {},
-    credentials: {},
-    route: {
-        params: {},
-    },
-};
-
-class SetPasswordPage extends Component {
```

Comment:
NAB, I appreciate the attempt to organize this by moving these files around. But it makes the code changes harder to review since we need to look at one file that has been removed completely and then another file that has been added - when actually there is a lot of overlap between them.

Ideally you can make changes like this so that they show up as a file rename instead of a deleted/created file pair that way we get a nice clean diff and can focus on what has actually changed instead of working hard to figure that out.

Reply:
For sure, I've bookmarked that SO you sent me and will make sure I do this in the future!